### PR TITLE
Venv installation

### PR DESCRIPTION
**Pull request by esoty6**
Added README.md with installation instructions
Added .gitignore
Added requirements.txt

### CODE REVIEW

File: charades-server/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+venv_*/
```
Comment (Krystian120413) [inline charades-server/.gitignore:1]: Let's add eof line

File: charades-server/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Installation
+
+Remember to use virtual environment.
+
+### Set virtual environment
+
+```python
+py -m venv venv_charades
```
Comment (Krystian120413) [inline charades-server/README.md:8]: Let's add instruction for Unix systems
`py` works only on Windows, in unix you have to use the `python3` command (or create alias)

**Review by Krystian120413 — APPROVED** [review]
Good work! 🙌 🎸

File: charades-server/README.md (hunk 2 above)
Comment (esoty6) [reply charades-server/README.md:8]: I think that worked on my WSL, but sure

**Review by Krystian120413 — APPROVED** [review]

**Pull request merged**